leakcanary-android-core/src/main/java/leakcanary/internal/InternalLeakCanary.kt

```diff
@@ -46,6 +44,13 @@ internal object InternalLeakCanary : (Application) -> Unit, OnObjectRetainedList
   var applicationVisible = false
     private set
 
+  private val isJunitAvailable = try {
```

Comment:
Is this val eagerly calculated (i.e before the Handler post called below) since it does not use `get() =` syntax?

Reply:
You're right, this should be a lazy.

Reply:
PR that fixes it: https://github.com/square/leakcanary/pull/1629

Reply:
Awesome. So quick 👊